### PR TITLE
Like and Comment layouts

### PR DESCRIPTION
**Pull request by divyayadav1606**

### CODE REVIEW

File: app/src/main/res/layout/post_actions.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageButton
```
Comment (deonna) [inline app/src/main/res/layout/post_actions.xml:7]: Can we name this `ibLike` for naming consistency?

File: app/src/main/res/layout/post_actions.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        android:layout_height="wrap_content"
+        android:background="@android:color/transparent"
+        android:src="@drawable/heart"
+        android:layout_margin="10dp"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true" />
+
+    <ImageButton
+        android:id="@+id/commentBtn"
```
Comment (deonna) [inline app/src/main/res/layout/post_actions.xml]: Can we name this `ibComment` for naming consistency?

File: app/src/main/res/layout/post_actions.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        android:layout_height="wrap_content"
+        android:background="@android:color/transparent"
+        android:src="@drawable/comment"
+        android:layout_marginTop="10dp"
+        android:layout_alignParentTop="true"
+        android:layout_toRightOf="@+id/likeBtn"
+        android:layout_alignBottom="@+id/likeBtn" />
+
+    <TextView
+        android:id="@+id/likesCnt"
```
Comment (deonna) [inline app/src/main/res/layout/post_actions.xml]: Can we name this `tvLikesCount` for naming consistency?

**Review by deonna — COMMENTED** [review]
Other than naming consistency comments, LGTM.

**Comment by divyayadav1606** [comment]
made review changes

**Pull request merged**